hvplot/converter.py

```diff
@@ -348,9 +348,17 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
 
         # Process options
         self.stacked = stacked
-        self._style_opts, kwds = self._process_style(kwds)
 
-        plot_opts = {**self._default_plot_opts,
+        try:
+            if self.kind in self._colorbar_types and not use_dask:
```

Comment:
`use_dask` is only effective for intake datasets, when operating directly on a dask dataframe or dask array backed xarray the data may still be out-of-core. I think that will need to be handled inside `_process_symmetric`.

Reply:
Does handling it mean skipping it?
```
            # chunks mean it's lazily loaded; nanquantile will eagerly load
            if self.data.chunks:
                return False
```